analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ParametersCorrectOrder.CSharp12.cs

```diff
@@ -1,24 +1,5 @@
 ﻿using System;
 
-namespace Tests.Diagnostics
```

Comment:
https://github.com/SonarSource/sonar-dotnet/pull/8093#discussion_r1342439023

Yes, it probably was a leftover of a copy-paste.
Removed.